packages/terra-clinical-header/src/Header.jsx

```diff
@@ -31,25 +30,36 @@ const defaultProps = {
 };
 
 const Header = ({ title, startContent, endContent, ...customProps }) => {
-  const titleElement = <h1 className={cx('title')}>{title}</h1>;
-  let headerContent;
-
-  if (startContent || endContent) {
-    headerContent = (
-      <Arrange
-        fitStart={startContent}
-        fitEnd={endContent}
-        fill={titleElement}
-        align="center"
-      />
+  let startElement;
+  if (startContent) {
+    startElement = <div className={cx('flexEnd')}>{startContent}</div>;
+  }
+
+  let titleElement;
+  if (title) {
+    titleElement = (
+      <div className={cx('flexFill')}>
+        <div className={cx('titleContainer')}>
+          <h1 className={cx('title')}>
+            {title}
+          </h1>
+        </div>
+      </div>
     );
-  } else {
-    headerContent = titleElement;
+  }
+
+  let endElement;
+  if (endContent) {
+    endElement = <div className={cx('flexEnd')}>{endContent}</div>;
   }
 
   return (
     <header {...customProps} className={cx('header', customProps.className)}>
-      {headerContent}
+      <div className={cx('flexContainer', customProps.className)}>
```

Comment:
It looks like customProps.className is getting spread onto both the outer and inner flex container. Would it simplify anything if the css from `flexContainer` was applied to `header` directly? Then the extra div could be removed.